apiserver/facades/agent/upgradeseries/upgradeseries.go

```diff
@@ -282,6 +282,24 @@ func (a *API) authAndMachine(e params.Entity, canAccess common.AuthFunc) (common
 	return a.GetMachine(tag)
 }
 
+// PinnedLeadership returns all pinned applications and the entities that
+// require their pinned behaviour, for leadership in the current model.
+func (a *API) PinnedLeadership() (params.PinnedLeadershipResult, error) {
```

Comment:
A shame that I didn't do this first, because then we wouldn't have had this implicitly exposed, requiring us to keep it in order to avoid facade bumping.